terraform/bigquery.tf

```diff
@@ -473,17 +473,17 @@ resource "google_bigquery_routine" "unique_events_by_date_type" {
     FROM
       `${google_bigquery_dataset.default.dataset_id}.${google_bigquery_table.raw_events_table.table_id}`
     WHERE
-      received >= start
-      AND received <= end
+      received >= startDate
```

Comment:
Nit: is this a date or a datetime? If it's a date, +1. If it's a datetime, maybe `startTimestamp`?

Reply:
Changed